app/config.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+from kombu import Exchange, Queue
+
+
+class Config():
+    NOTIFICATION_QUEUE_PREFIX = os.getenv('NOTIFICATION_QUEUE_PREFIX')
+    QUEUE_NAME = 'govuk-alerts'
+
+    # Logging
+    DEBUG = False
+    LOGGING_STDOUT_JSON = os.getenv('LOGGING_STDOUT_JSON') == '1'
+
+    NOTIFY_APP_NAME = 'govuk-alerts'
+    NOTIFY_AWS_REGION = 'eu-west-1'
+    NOTIFY_LOG_PATH = os.getenv('NOTIFY_LOG_PATH', 'application.log')
+
+    BROADCASTS_AWS_ACCESS_KEY_ID = os.getenv("BROADCASTS_AWS_ACCESS_KEY_ID")
+    BROADCASTS_AWS_SECRET_ACCESS_KEY = os.getenv("BROADCASTS_AWS_SECRET_ACCESS_KEY")
+    BROADCASTS_AWS_REGION = 'eu-west-2'
+    GOVUK_ALERTS_BUCKET_NAME = os.getenv("GOVUK_ALERTS_BUCKET_NAME")
+
+    FASTLY_SERVICE_ID = os.getenv("FASTLY_SERVICE_ID")
+    FASTLY_API_KEY = os.getenv("FASTLY_API_KEY")
+    FASTLY_SURROGATE_KEY = 'notify-emergency-alerts'
+
+    CELERY = {
+        'broker_url': 'sqs://',
+        'broker_transport_options': {
+            'region': NOTIFY_AWS_REGION,
+            'visibility_timeout': 310,
+            'queue_name_prefix': NOTIFICATION_QUEUE_PREFIX,
+            'wait_time_seconds': 20,  # enable long polling, with a wait time of 20 seconds
```

Comment:
we want this, or things will paradoxically take longer as long polling won't be enabled.

20 means "celery will send https request to SQS. SQS will hold http connection open for 20 seconds. if a task comes in, it will immediately return that in a response."